chromadb/migrations/migrate_collections.py

```diff
@@ -0,0 +1,103 @@
+# This file contains functions for migrating data from one version of Chroma to another.
+from typing import Optional, Union
+from chromadb.api.client import Client
+from chromadb.api.configuration import (
+    Configuration,
+    ConfigurationParameter,
+)
+from chromadb.segment.impl.vector.hnsw_params import HnswParams
+from pypika import Table
+from chromadb.db.base import ParameterValue, get_sql
+from tqdm import tqdm
+
+
+def migrate_collections(
```

Comment:
do we want to version these? how does this extend in the future?

Reply:
Let's discuss. My thought here was we would wait for the next one to happen and then write the right abstraction, but you make the point elsewhere that we do already have other migrations. Will take a look at those.